src/openrct2/ride/TrackPaint.h

```diff
@@ -399,7 +399,10 @@ constexpr bool kDiagSpriteMap[kNumOrthogonalDirections][4] = {
 
 extern const uint8_t mapLeftEighthTurnToOrthogonal[5];
 
-extern const size_t MiniGolfPeepAnimationLengths[];
+namespace OpenRCT2
+{
+    extern const size_t MiniGolfPeepAnimationLengths[];
+}
```

Comment:
Hmm, I really don't like how these are just declared there, far away what is now `Vehicle.MiniGolf.cpp`. I realise you've only put them in a namespace, but could you move them to `VehiclePaint.h` so they're declared in the same folder at least?